stats/tinymembench

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+LKP_SRC = ENV['LKP_SRC'] || File.dirname(File.dirname(File.realpath($PROGRAM_NAME)))
+
+require "#{LKP_SRC}/lib/log"
+
+stats = Hash.new { |h, k| h[k] = [] }
+
+while (line = $stdin.gets)
+  case line
+  when /(standard mem|MOVSB|MOVSD|SSE2)/
```

Comment:
thanks, help comment like # SSE2 copy                                            :  17668.2 MB/s (0.5%)
so easier to know how the line looks like, and what below code tries to parse

Reply:
Added comment for the same.